docs.json

```diff
@@ -113,7 +113,7 @@
                   {
                     "group": "OAuth",
                     "pages": [
-                      "embedded-wallets/authentication/login/oauth/oauth-implementation",
```

Comment:
I do agree it's cleaner, but can we leave this for now? Changing this will break the redirect implemented in the magic-site repo.